frontend/src/app/lib/router.js

```diff
@@ -1,11 +1,6 @@
 /* global ga */
```

Comment:
It can! There's actually an argument that it belongs in `/components/App.js`, I'll move it there.